data/spiders/rips/addiction.json

```diff
@@ -0,0 +1,9232 @@
+[
```

Comment:
Why do you need to store these files in git? Shouldn't your spider fetch them dynamically?

Reply:
https://github.com/littlepea/safeway-data/pull/18/commits/0a48f991ef028a8da748bd9c6c31cd4003080235